docs/config.cue

```diff
@@ -25,7 +25,7 @@ params: {
 	github_project_repo: "https://github.com/hofstadter-io/hof"
 	github_repo:         "https://github.com/hofstadter-io/hof"
 	time_format_default: "January 2, 2006"
-	keywords:            "hof hofstadter documentation docs"
+	keywords:            ["hof", "hofstadter", "documentation", "docs"]
```

Comment:
After reading a bit of that Google SEO, I'm not sure about the duplication or stuffing here, but let's leave it for now